src/migrateUIFolder.test.ts

```diff
@@ -17,20 +18,46 @@ const hasRecord = (contentRecord: ContentRecord, recordId: string): boolean =>
     (child, childId) => childId === recordId || hasRecord(child, recordId)
   );
 
+jest.mock(`./generateId`, () => {
+  let counter = 0;
+  return {
+    generateId: jest.fn(() => {
+      const id = `00${counter}`;
+      counter += 1;
+
+      return id;
+    }),
+  };
+});
+
 describe("migrateUIFolder", () => {
-  it("returns a valid ActiveUI5 /ui folder on a small input", () => {
-    const migratedUIFolder = migrateUIFolder(smallLegacyUIFolder, servers);
+  it("returns a valid ActiveUI5 /ui folder on a small input", async () => {
+    const migratedUIFolder = await migrateUIFolder(
+      smallLegacyUIFolder,
+      servers
+    );
     expect(migratedUIFolder).toMatchSnapshot();
   });
 
-  it("returns a valid ActiveUI5 /ui folder on a real life input", () => {
-    const migratedUIFolder = migrateUIFolder(legacyUIFolder, servers);
+  it("returns a valid ActiveUI5 /ui folder on a real life input", async () => {
+    const migratedUIFolder = await migrateUIFolder(legacyUIFolder, servers);
+    expect(migratedUIFolder).toMatchSnapshot();
+  });
+
+  it("returns a valid ActiveUI5 /ui folder that includes calculated measures when the input includes a pivotFolder", async () => {
+    const migratedUIFolder = await migrateUIFolder(
```

Comment:
I cannot see the diff on GitHub, it's too big.
This makes me think that nobody will go through the whole snapshot anyway.
So I propose to narrow down the target of the test: extract the `calculated_measures` folder from `migratedUIFolder`, and only snapshot that.